content/docs/concurrent-mode-suspense.md

```diff
@@ -583,9 +583,9 @@ function ProfileTimeline({ resource }) {
 }
 ```
 
-**[Try it on CodeSandbox](https://codesandbox.io/s/infallible-feather-xjtbu)**
+**[Experimente no CodeSandbox](https://codesandbox.io/s/infallible-feather-xjtbu)**
 
-In the previous Suspense example, we only had one `resource`, so we held it in a top-level variable. Now that we have multiple resources, we moved it to the `<App>`'s component state:
+No exemplo anterior utilizando Suspense, tínhamos apenas um `recurso`, portanto mantivemos em uma variável de nível superior. Agora que temos vários recursos, o movemos para o state do componente `<App>`:
```

Comment:
Acredito que `resource` não deve ser traduzido, já que é um recurso do código acima, o que acha? Também vi em outros lugares ele ter sido traduzido, se for corrigir isso dê uma pesquisada no restante da página.